pkg/codegen/nodejs/gen_crd2pulumi.go

```diff
@@ -7,47 +7,48 @@ import (
 	"github.com/pulumi/pulumi/pkg/v2/codegen/schema"
 )
 
-func GenerateTypes(tool string, pkg *schema.Package) (string, error) {
+// GenCRDTypes returns a buffer containing the code for header, imports,
+// namespaces, and interfaces of all versions.
+func GenCRDTypes(tool string, pkg *schema.Package) (*bytes.Buffer, error) {
```

Comment:
What was the reason for changing `string` to `*bytes.Buffer`? Just curious.

Reply:
`go/gen_crd2pulumi.go` returned a `map[string]*bytes.Buffer`, so to be a bit consistent I changed this one from `string` to `*bytes.Buffer`. Also, the generated types string would be concatenated with the generated classes strings, so just writing to 1 buffer instead should be slightly more efficient.